traefik/hub-crds/kustomization.yaml

```diff
@@ -1,12 +0,0 @@
----
```

Comment:
I don't agree with this commit. People not using hub doesn't need to install CRDs.